diff --git a/java/src/main/java/me/zed_0xff/zombie_buddy/Loader.java b/java/src/main/java/me/zed_0xff/zombie_buddy/Loader.java
index 2f3fb31..aa5b90b 100644
--- a/java/src/main/java/me/zed_0xff/zombie_buddy/Loader.java
+++ b/java/src/main/java/me/zed_0xff/zombie_buddy/Loader.java
@@ -555,7 +555,7 @@ public class Loader {
         "ZModUnbork"
     );
 
-    private static void autoFixModOrder(ArrayList<String> mods) {
+    private static void autoFixModOrder(List<String> mods) {
         if (Utils.isBlank(mods)) {
             return;
         }
@@ -586,14 +586,14 @@ public class Loader {
         return out;
     }
 
-    private static void moveModToIndex(ArrayList<String> mods, String modId, int index) {
+    private static void moveModToIndex(List<String> mods, String modId, int index) {
         int oldIndex = mods.indexOf(modId);
         if (oldIndex < 0) return;
         mods.remove(oldIndex);
         mods.add(Math.min(index, mods.size()), modId);
     }
 
-    public static void loadMods(ArrayList<String> mods) {
+    public static void loadMods(List<String> mods) {
         if (g_config.auto_fix_mod_order()) {
             autoFixModOrder(mods);
         }
diff --git a/java/src/main/java/me/zed_0xff/zombie_buddy/patches/Patch_ZomboidFileSystem.java b/java/src/main/java/me/zed_0xff/zombie_buddy/patches/Patch_ZomboidFileSystem.java
index d75b33c..7b738b7 100644
--- a/java/src/main/java/me/zed_0xff/zombie_buddy/patches/Patch_ZomboidFileSystem.java
+++ b/java/src/main/java/me/zed_0xff/zombie_buddy/patches/Patch_ZomboidFileSystem.java
@@ -32,7 +32,7 @@ public class Patch_ZomboidFileSystem {
     @Patch(className = "zombie.ZomboidFileSystem", methodName = "loadMods")
     public class Patch_loadMods2 {
         @Patch.OnEnter
-        public static void enter(ArrayList<String> mods, @Patch.Local("t0") long t0) {
+        public static void enter(List<String> mods, @Patch.Local("t0") long t0) {
             Logger.info("ZomboidFileSystem.loadMods(" + mods.size() + " mods) ...");
             long loaderStartNs = System.nanoTime();
             Loader.loadMods(mods);
@@ -42,7 +42,7 @@ public class Patch_ZomboidFileSystem {
         }
 
         @Patch.OnExit
-        public static void exit(ArrayList<String> mods, @Patch.Local("t0") long t0) {
+        public static void exit(List<String> mods, @Patch.Local("t0") long t0) {
             long elapsedMS = (System.nanoTime() - t0) / 1_000_000L;
             if ( elapsedMS > 1000 ) Logger.info("ZomboidFileSystem.loadMods(" + mods.size() + " mods) took " + elapsedMS + " ms");
         }
